src/managed/Microsoft.Extensions.DependencyModel/JsonTextReaderExtensions.Netstandard2_0.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+
+namespace Microsoft.Extensions.DependencyModel
+{
+    internal static class JsonTextReaderExtensions
+    {
+        internal static bool TryReadStringProperty(this ref Utf8JsonReader reader, out string name, out string value)
+        {
+            name = null;
+            value = null;
+            if (reader.Read() && reader.TokenType == JsonTokenType.PropertyName)
+            {
+                name = reader.GetStringValue();
+                
+                if (reader.Read())
+                {
+                    if (reader.TokenType == JsonTokenType.String)
+                    {
+                        value = reader.GetStringValue();
+                    }
+                    else
+                    {
+                        reader.Skip();
+                    }
+                }
+
+                return true;
+            }
+
+            return false;
+        }
+
+        internal static void Skip(this ref Utf8JsonReader jsonReader)
```

Comment:
There is an issue to add this API: https://github.com/dotnet/corefx/issues/33295